KS.Fiks.IO.Arkiv.Client/ForenkletArkivering/ArkivmeldingFactory.cs

```diff
@@ -111,14 +113,16 @@ public static Arkivmelding GetArkivmelding(OppdaterSaksmappe input)
                             SkjermingDokument = "Hele"
                         };
                     }
+
+                    var filnavn = input.nyUtgaaendeJournalpost.hoveddokument.filnavn;
                     
                     var dok = new Dokumentobjekt
                     {
-                        Format = "X", //TODO Her mangler noe eller det er noe som ikke stemmer.
-                        Variantformat = "X", //TODO Her mangler noe eller det er noe som ikke stemmer.
-                        Versjonsnummer = "1", //TODO Her mangler noe eller det er noe som ikke stemmer.
-                        Filnavn = "", //TODO Her mangler noe eller det er noe som ikke stemmer.
-                        ReferanseDokumentfil = input.nyUtgaaendeJournalpost.hoveddokument.filnavn
+                        Format = filnavn.Substring(filnavn.LastIndexOf('.')),
+                        Variantformat = VariantformatDefault,
+                        Versjonsnummer = VersjonsnummerDefault, 
+                        Filnavn = filnavn,
+                        ReferanseDokumentfil = input.nyUtgaaendeJournalpost.hoveddokument.referanseDokumentFil
```

Comment:
Antagelig noe som vil være satt av arkivet for oppslag. Foreslår at vi tar en runde på denne på torsdag.